docs/self-hosted-runner/self_hosted_runner.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
```

Comment:
Instead of committing a patch, would be better to apply this directly and commit the result